apache/rocketmq/v2/admin.proto

```diff
@@ -38,6 +43,105 @@ message ChangeLogLevelRequest {
 
 message ChangeLogLevelResponse { string remark = 1; }
 
+message DescribeTopicStatusRequest {
+  optional string instance_id = 1;
+  Resource topic = 2;
+}
+
+message DescribeTopicStatusResponse {
+  Status status = 1;
+  MessageType topic_message_type = 2;
+  string description = 3;
+  google.protobuf.Timestamp create_timestamp = 4;
+  map<string, string> tags = 5;
+}
+
+message DescribeTopicSubscriptionRequest {
+  optional string instance_id = 1;
+  Resource topic = 2;
+}
+
+message DescribeTopicSubscriptionResponse {
+  message SubscriptionInfo {
+    Resource group = 1;
+    FilterExpression expression = 2;
+    bool online = 3;
+    google.protobuf.Timestamp last_update_timestamp = 4;
+  }
+  Status status = 1;
+  repeated SubscriptionInfo subscriptionInfo = 2;
+}
+
+message DescribeConsumerAccumulateRequest {
+  optional string instance_id = 1;
+  Resource group = 2;
+  repeated Resource topics = 3;
```

Comment:
topics may not be necessary because brokers should have this knowledge

Reply:
Group can subscribe to multiple topics, if not specified, return all the topics' accumulation.